Tribler/Core/Utilities/utilities.py

```diff
@@ -244,3 +244,47 @@ def fix_torrent(file_path):
         fixed_data = bencode(fixed_data)
 
     return fixed_data
+
+def translate_peers_into_health(peer_info_dicts):
+    """
+    peer_info_dicts is a peer_info dictionary from LibTorrentDownloadImpl.create_peerlist_data
+    purpose : where we want to measure a swarm's health but no tracker can be contacted
+    """
+    num_seeders = 0
+    num_leech = 0
+
+    num_all_peer = len(peer_info_dicts)
```

Comment:
I don't think it's necessary to create a variables just to hold the length of this dict since `num_all_peer` is used only once.